swig/python/test_comedi_command.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
```

Comment:
This seems to be more than a simple rename, as the new file is twice as long as the old one. A better commit message is needed.